importer/api/__init__.py

```diff
@@ -148,7 +148,7 @@ def import_children(course, element, parent, parent_dpath):
         None
     """
     # pylint: disable=too-many-locals
-    title = element.attrib.get("display_name", "MISSING")
+    title = element.attrib.get("display_name", "...")
```

Comment:
It should be: in theory all the code we wrote should be unicode safe: if it's not, we have a bug that we have to address, so maybe it's a good idea to use this case to verify everything.